portal/templates/admin/patients_substudy.html

```diff
@@ -81,11 +81,11 @@ <h2>{{list_title}}</h2>
                       <td>{{patient.assessment_status if patient.assessment_status}}</td>
                       <td>{{patient.current_qb if patient.current_qb}}</td>
                       <td>
-                        {% if patient.action_state in ("Required", "Due", "Overdue") %}
-                          <a href='{{url_for("patients.patient_profile", patient_id=patient.id, _anchor="postInterventionQuestionnaireLoc")}}'>{{patient.action_state}}</a>
-                        {% else %}
+                        {%- if patient.action_state in ("Required", "Due", "Overdue") -%}
```

Comment:
I believe the extra minus signs near the % will strip the contained white space.  Is that intentional or needed?  Also on lines 86 and 88 below...

Reply:
@pbugni yes, it is to get rid of the white space and new line characters that I see being rendered for each of that table cell.